diesel-factories-code-gen/src/lib.rs

```diff
@@ -41,6 +52,129 @@ struct DeriveData {
     tokens: TokenStream,
 }
 
+trait MonkeyPathSegment {
```

Comment:
I've seen other crates use the pattern `*Extension` for "monkey patching traits". Diesel for example has [`OptionalExtension`](http://docs.diesel.rs/diesel/result/trait.OptionalExtension.html#tymethod.optional).

What do you think about naming these "monkey" traits like that? So this one would become `PathSegmentExtension`.

Reply:
Happy to switch it if that's the standard. Definitely feels less hacky when we call it extension 😄